src/components/TimeCardPage/SubmitCard.js

```diff
@@ -6,15 +6,27 @@ import Button from 'react-bootstrap/Button';
 
 
 
+
 export default function SubmitCard(props) {
 
     const [submitted, setSubmitted] = useState(false); 
     const [submitDate, setSubmitDate] = useState(null); 
+    const [state, setState] = useState('Rejected'); 
+    const [rejected, setRejected] = useState(true);
+    const [report_type, setReport] = useState('this is rejected'); 
+
+    const model = {
+        AuthorUUID: "XXXX",
+        Type: "Report / Comment, etc",
+        Timestamp: "",
+        Content: "invalid hours"
+    }
+
 
     useEffect(() => {
         //TODO - API Call to determine if the table has been submitted or not.
         //Will set submitted? here and also submitDate if it was submitted to grab the date 
-        
+        setReport(model.Content)        
```

Comment:
I'd actually prefer to pass this data down from `TimeSheet.js` so that we only have one API call getting the data. The `apiClient.getUserTimesheets` call in `TimeSheet.js` will get this model for you (after it's added to the endpoint), so then you can pass it to `SubmitCard` as a prop, ex.
```jsx
<SubmitCard comment={selectedTimesheet.comment} />
```
Then here in this file you can get it as `props.comment` and use it for styling/conditional rendering/rendering below

Reply:
This is what the comment object would look like in this case from the .comment field 

`"Comment":{
        "AuthorUUID":"XXXX", 
        "Type":"Report / Comment, etc", 
        "Timestamp":"", 
        "Content":":)" 
    }}`

It would be helpful to define this as a constant  / test object and pass it into props for now using what Sofie described above - let me know if you need clarification with this!